docs.json

```diff
@@ -2461,6 +2461,18 @@
       "source": "/library/init",
       "destination": "/models/ref/python/functions/init"
     },
+    {
```

Comment:
This doesn't actually do anything. Leaving it here just to show Mintlify what I tried, but I'll remove these redirects before merging this PR.